mycroft/configuration/mycroft.conf

```diff
@@ -470,10 +470,11 @@
     },
 
     "wake_up": {
-        "module": "ovos-ww-plugin-pocketsphinx",
-        "phonemes": "W EY K . AH P",
-        "threshold": 1e-20,
-        "lang": "en-us",
+        "module": "ovos-ww-plugin-vosk",
```

Comment:
needs to be added to requirements.txt and stt extra requirements

Reply:
I think I put the appropriate versions